Kickstarter-iOS/Views/Cells/PledgeDescriptionCell.swift

```diff
@@ -123,12 +136,26 @@ final class PledgeDescriptionCell: UITableViewCell, ValueCell {
     _ = (views, self.descriptionStackView)
       |> ksr_addArrangedSubviewsToStackView()
 
-    self.descriptionStackView.setCustomSpacing(10.0, after: self.dateLabel)
-
     _ = ([self.descriptionStackView], self.rootStackView)
       |> ksr_addArrangedSubviewsToStackView()
   }
 
+  private func sizeAndTransformRewardCardView() {
+    self.rewardCardContainerView.layoutIfNeeded()
+
+    let (actualSize, thumbnailSize) = rewardCardViewSizes(with: self.rewardCardContainerView)
+
+    guard thumbnailSize.width > 0, thumbnailSize.height > 0 else { return }
+
+    self.rewardCardContainerMaskViewWidthConstraint?.constant = thumbnailSize.width
+    self.rewardCardContainerMaskViewHeightConstraint?.constant = thumbnailSize.height
+
+    self.rewardCardContainerView.transform = transformFromRect(
```

Comment:
Good q - so the view is actually rendered at full size and then has a `transform` applied to make it fit within the 'thumbnail' size. If we constrain all edges, it will cause the view to squish up against itself thinking that it in fact needs to be that size.